cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/goals/PreferredLeaderElectionGoal.java

```diff
@@ -63,39 +63,37 @@ private void sanityCheckOptimizationOptions(OptimizationOptions optimizationOpti
     }
   }
 
+  private boolean skipOperationOnURP(TopicPartition tp, String operation) {
```

Comment:
nit: use shouldSkipOperationOnURP is slightly better based on naming convention.

Reply:
this is better. making the change